services/offline/src/index.js

```diff
@@ -0,0 +1,3 @@
+export { OfflineProvider } from './lib/offline-provider'
+export { CacheableSection, useCacheableSection } from './lib/cacheable-section'
+export { useOnlineStatus } from './lib/online-status'
```

Comment:
`OfflineProvider` will be used in the App Adapter to wrap an app and provide relevant context;
the other 3 are intended to be used in aps